recipe/recipe.yaml

```
@@ -0,0 +1,105 @@
+context:
+  version: '1.20.0'
+
+# Note: This recipe is specifically designed to work well with the autotick bot.
+# Also refer to https://github.com/conda-forge/rust-feedstock/blob/main/recipe/meta.yaml.
+package:
+  name: ${{ polars_variant }}
+  version: ${{ version }}
+
+source:
+  - if: polars_variant == 'polars'
+    then:
+      url: https://pypi.org/packages/source/p/polars/${{ polars_variant | replace('-', '_') }}-${{ version }}.tar.gz
```

The autotick-bot cannot read variant variables here.

```suggestion
      url: https://pypi.org/packages/source/p/polars/polars-${{ version }}.tar.gz
```

---

the pathway for adding support for your original proposal is here: https://github.com/prefix-dev/rattler-build-conda-compat/pull/66